Add docs for placement rules in SQL (#6618)
pingcap/docs: system-variables.md

>
> Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.

### placement_checks

- Scope: SESSION | GLOBAL
- Default value: ON

> - This variable controls whether DDL validates that [Placement Rules in SQL](/placement-rules-in-sql.md) are valid when creating tables.

Reviewer: Not sure whether my comment is correct. PTAL
```suggestion
- This variable controls whether a DDL execution validates [Placement Rules in SQL](/placement-rules-in-sql.md) when a table is created.
```

Author: How about "ddl statements" instead of DDL execution?

Reviewer: Modified.

Author: OK. I've made further small edits (DDL statements implies "when creating tables", since that is a DDL statement).
